Dockerfiles/manifests/agent-kubelet-only.yaml

```diff
@@ -36,8 +38,11 @@ spec:
             memory: "256Mi"
             cpu: "200m"
         livenessProbe:
-          exec:
-            command:
-            - ./probe.sh
+          httpGet:
+            path: /live
```

Comment:
Could we use `health` instead of `live`?

Reply:
Right now the handler replies in any url (I just test with `curl localhost:5555`)
The `/live` path is to allow to later separate the liveness probe and the readyness probe, if needed